dll/win32/shell32/droptargets/CexeDropHandler.cpp

```diff
@@ -84,6 +84,8 @@ HRESULT WINAPI CExeDropHandler::Drop(IDataObject *pDataObject, DWORD dwKeyState,
 
             pszSrcList += wcslen(pszSrcList) + 1;
         }
+
+        GlobalUnlock(medium.hGlobal);
```

Comment:
In the previous function you fixed there was a `ReleaseStgMedium(&medium);` just after the GlobalUnlock you added. But not here. Why?

Reply:
Oversight. Since resolved.